blake2b/blake2b.go

```diff
@@ -0,0 +1,71 @@
+// Package blake2b is a thin wrapper over golang.org/x/crypto/blake2b,
+// and additionally provides a BLAKE2b-128-concat hashing algorithm.
+package blake2b
+
+import (
+	"hash"
+
+	"golang.org/x/crypto/blake2b"
+)
+
+// State for the blake2b_128_concat hasher
+type state struct {
```

Comment:
Please rename this for clarity:

```suggestion
type concatState struct {
```

Reply:
fixed in latest upload